### PR TITLE
Space Traveler Hub

### PR DESCRIPTION
**Pull request by ahmetbozaci**
### In this project,

[Live link](https://space-traveler-hub.netlify.app/)
[Readme](https://github.com/ahmetbozaci/space-traveler-hub/blob/develop/README.md)

1.  We create 3 pages,  `rockets`, `mission`, and `my profile.` The main page is the `rockets page`. 
2.  Use `react-router  v6`.  When clicking the page in the navigation bar that page is opening.
3.  Rockets and missions page include: 
       - Rockets from [API](https://api.spacexdata.com/v3/rockets)
       - Missions from [API](https://api.spacexdata.com/v3/missions)
       - Every rocket and mission  include a  button for sending API new key, value `reserved: true` also it is work reserve `reserved: false`
       - If `reserved: true`  `not a  member` **badge** change to `active member` **badge** on missions page or add  `reserved` **badge** on rockets page.
4. My profile page includes:
   - Missions and rockets which include `reserved: true`
   - Also this section includes the `leave` or `cancel` button for making `reserved: false`
   - Mission section includes an extra `read more` button that is opening a Wikipedia link about the mission.
   - Optional parts added to the kanban board.
5. Page styled with `react-bootstrap` so we make it responsive.

### CODE REVIEW

File: src/redux/rocketReducer/RocketReducer.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { RocketConstants } from './RocketAction';
+
+const initialState = {
+  rockets: [],
+};
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (bimbolabuari) [inline src/redux/rocketReducer/RocketReducer.js]: Kindly remove disabled Linters, Linting improves readability, removes errors before execution and also make your code more professional ✊ 

Please check for other occurences.

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi @ahmetbozaci and @VICTOR-CHUKWU , 

You have done a good job so far 🚀  

## Highlights 🎉 
- The Space Travelers' Hub consists of Rockets, Missions, and the My Profile section. 💯 
-  Profile section displays all reserved rockets and space missions ✊ 
-  The Missions section displays a list of current missions along with their brief description and participation status 👍 
- There is a button next to each mission that allows users to join the selected mission or leave the mission the user joined earlier 🎉 
-  The Rockets section displays a list of all available SpaceX rockets. Users can book each rocket by clicking the reservation button or cancel the previously made booking 🚀 

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

**Comment by VICTOR-CHUKWU** [comment]
I have removed the  linters

**Review by DeliceLydia — APPROVED** [review]
# Congratulations! 🎉🎉 

## Hi Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**